shared/video-sdk/develop/real-time-transcription/project-implementation/ios.mdx

```diff
@@ -55,13 +55,13 @@ You see errors in your IDE. This is because the layout refers to methods that yo
     let instanceID = "Test-1"
     var authorizationHeader = ""
     var builderToken = ""
-    var speechToTextTaskId = ""  // Returned by the start method
-    var isSpeechToTextRunning: Bool = false
+    var realTimeTranscriptionTaskId = ""  // Returned by the start method
+    var isRealTimeTranscriptionRunning: Bool = false
     ```
 
-### Manage a Real-Time Transcription task
+### Manage a real-time transcription task
 
-Take the following steps to start, stop and query a text-to-speech task in your <Vpl k="CLIENT" />.
+Take the following steps to start, stop and query a text-to-real-time transcription task in your <Vpl k="CLIENT" />.
```

Comment:
Please double check , if this is correct as for rest other doc under "Manage a Real-Time Transcription task", 
it is always " speech-to-text"   not "text-to-speech". In case of typo  please change 
text-to-real-time transcription --> real-time transcriptio